cardano-testnet/test/Test/FoldBlocks.hs

```diff
@@ -35,13 +35,12 @@ instance Show FoldBlocksException where
 -- that main thread blocks on.
 prop_foldBlocks :: H.Property
 prop_foldBlocks = H.integrationRetryWorkspace 2 "foldblocks" $ \tempAbsBasePath' -> do
-
   -- Start testnet
   base <- HE.noteM $ liftIO . IO.canonicalizePath =<< HE.getProjectBase
-  configTemplate <- H.noteShow $ base </> "configuration/defaults/byron-mainnet/configuration.yaml"
+  configTemplate <- H.noteShow $ base </> "configuration/defaults/byron-mainnet/configuration.yaml" --- outside of the package directory !!
```

Comment:
> outside of the package directory !!

Yeah, that's what `base` and `getProjectBase` means.  Is there a way to make this clearer without adding a comment?

Reply:
That comment is a TODO for me.
I think it is better to move copies of those file to the package itself
or maybe there is yet an other way to do this.